spec/factories/notes.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :note do
+    note { "MyText" }
+    contact { nil }
```

Comment:
I think it'd be better, if factory bot generates a note with a contact by default (as `belongs_to` is required by default). For this to happen, we need to put `contact` at the beginning: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations

Reply:
thanks - I've updated.